query-string/index.d.ts

```diff
@@ -13,7 +13,7 @@ declare module "query-string" {
      * Leading ? or # are ignored, so you can pass location.search or location.hash directly.
      * @param str
      */
-    export function parse(str: string): { [key: string]: string | string[] };
+    export function parse(str: string): { [key: string]: string | string[] | null };
```

Comment:
`undefined` is also a valid value, so we should add `| undefined` here as well. So not sure if these changes actually add anything extra.